pkg/loader/load.go

```diff
@@ -1008,3 +1008,28 @@ func isSetTiFlashReplica(sql string) bool {
 
 	return false
 }
+
+func isSetTiFlashMode(sql string) bool {
```

Comment:
Can we merge this function into `isSetTiFlashReplica`? They look so similar except for L1029

Reply:
Yes, I think we can merge the function.